extensions/wikia/WallNotifications/WallNotificationsEveryone.class.php

```diff
@@ -359,8 +402,8 @@ public function clearQueue( $onlyCache = false ) {
 		// TODO: performance of this queries
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)